src/containers/BooksList.js

```diff
@@ -0,0 +1,33 @@
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+/* eslint-disable no-unused-vars */
+import Book from '../components/Book';
+
+const BooksList = ({ books }) => {
+  const allBook = books.map(book => (<Book key = {`book-${book.bookId}`} book = {book} />));
+
+  return (
+    <div className = "book-show-div">
+     <table className = "book-table">
+       <thead>
+         <tr className = "book-row">
+          <th className = "book-header">BookId</th>
+          <th className = "book-header">title</th>
+          <th className = "book-header">category</th>
+         </tr>
+       </thead>
+       <tbody>
+         {allBook}
+       </tbody>
+     </table>
+    </div>
+  );
+};
+
+BooksList.propTypes = {
+  books: PropTypes.arrayOf.isRequired,
```

Comment:
- [ ] I got another console error from `arrayOf` here:

![Screenshot from 2020-12-29 12-15-16](https://user-images.githubusercontent.com/4246637/103276547-89917000-49cf-11eb-8a40-c6f03a004e77.png)

Same concept `arrayOf` should accept a proptype argument. We can use:

```js
books: PropTypes.arrayOf(PropTypes.object).isRequired,
```